cmd/humanInsertion/humanInsertion.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/maf"
+	"log"
+)
+
+func humanInsertion(mafFile string, outBed_ins string, outBed_del string, species_ins string, species_del string) {
+	//initialize variables
+	mafRecords := maf.Read(mafFile) //Read entire mafFile. mafRecords has type Maf
```

Comment:
I'm try to wrap my head around type Maf and was wondering if someone can tell me what the "S", "I", and "E" stand for in MafSLine, MafILine, and MafELine?

Reply:
That's the letter that begins the line. In a maf file, within each alignment block, each species has at least 1 line describing its status in the alignment block. For example, if the line for human starts with s, that means the human sequence is aligned; if the line for chimp starts with e, that means the chimp sequence is "empty" or does not align
Official description of the Maf format: https://genome.ucsc.edu/FAQ/FAQformat.html#format5